packages/meteor-base/package.js

```diff
@@ -25,12 +25,6 @@ Package.onUse(function(api) {
     'ddp',
     'livedata', // XXX COMPAT WITH PACKAGES BUILT FOR 0.9.0.
 
-    // These packages use the user agent of each incoming HTTP request to
-    // decide whether to inject <script> tags into the <head> of the
-    // response document to polyfill Web Sockets and/or ES5 support.
-    'sockjs-shim',
-    'es5-shim',
```

Comment:
I should not have removed `es5-shim` here, since the package still exists (unlike `sockjs-shim`), and has no impact on modern browser bundles.